sdk/ai/azure-ai-inference/README.md

```diff
@@ -451,6 +459,89 @@ TBD
 To generate embeddings for additional phrases, simply call `client.embed` multiple times using the same `client`.
 -->
 
+## Tracing
```

Comment:
I would put this section at the very bottom of the readme, since this is not really a "quickstart" kind of thing, but advanced.

Reply:
moved after troubleshooting